vim25/soap/client.go

```diff
@@ -198,10 +198,10 @@ func (c *Client) NewServiceClient(path string, namespace string) *Client {
 
 	client := NewClient(u, c.k)
```

Comment:
Should we pass in the existing transport, perhaps via a new constructor, instead of instantiating a new one that we will just toss?